docs/dev_guide/scale.md

```diff
@@ -0,0 +1,101 @@
+--- 
```

Comment:
This should also be on the docs website, in a dedicated section for Scalability.